docs/how-to-guides/adding-a-new-offline-store.md

```diff
@@ -30,6 +30,8 @@ There are two methods that deal with reading data from the offline stores`get_hi
 * `pull_latest_from_table_or_query` is invoked when running materialization (using the `feast materialize` or `feast materialize-incremental` commands, or the corresponding `FeatureStore.materialize()` method. This method pull data from the offline store, and the `FeatureStore` class takes care of writing this data into the online store.
 * `get_historical_features` is invoked when reading values from the offline store using the `FeatureStore.get_historical_features()` method. Typically, this method is used to retrieve features when training ML models.
 * `pull_all_from_table_or_query` is a method that pulls all the data from an offline store from a specified start date to a specified end date.
```

Comment:
might call out that this method is optional since it is only used to work with SavedDatasets

Reply:
I think you commented on the wrong method , you mean write_logged_features right?

Reply:
nope. pull_all_from_table_or_query right now in our logic is only exposed for SavedDatasets afaict